packages/guides/tests/unit/Compiler/CompilerContextTest.php

```diff
@@ -15,19 +15,22 @@
 
 use Doctrine\Deprecations\PHPUnit\VerifyDeprecations;
 use phpDocumentor\Guides\Nodes\ProjectNode;
+use PHPUnit\Framework\Attributes\WithoutErrorHandler;
 use PHPUnit\Framework\TestCase;
 
 final class CompilerContextTest extends TestCase
 {
     use VerifyDeprecations;
 
+    #[WithoutErrorHandler]
```

Comment:
Why is this necessary? The test passes when I remove it.

Reply:
see https://github.com/doctrine/deprecations/pull/66#issuecomment-2509771433